cloudify_hostpool/rest/service.py

```diff
@@ -0,0 +1,67 @@
+########
```

Comment:
i though we agreed to leave this file out of the merge for now. since it has no real implementation. didn't we?